redis/commands/core.py

```diff
@@ -1504,6 +1504,29 @@ def bitfield(
         """
         return BitFieldOperation(self, key, default_overflow=default_overflow)
 
+    def bitfield_ro(
+        self: Union["Redis", "AsyncRedis"],
+        key: KeyT,
+        encoding: str,
+        offset: BitfieldOffsetT,
```

Comment:
From the [docs](https://redis.io/commands/bitfield_ro/) it would appear that the offset ([GET encoding offset...]) section supports more items. Thoughts?

Reply:
Yes. I already have `items` parameter which accepts optional list of encoding/offset pairs

Reply:
Ugz, apologies, I literally didn't see that. My eyes cut off at offset. Ignore this comment